.github/workflows/diff-change-to-dist.yaml

```
@@ -52,19 +52,30 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const { readFile } = require('node:fs/promises')
-            const { comment } = await import('${{ github.workspace }}/.github/workflows/scripts/comments.mjs')
-
-            // Read diff from previous step
-            const diffText = await readFile('${{ github.workspace }}/dist.diff', 'utf8')
+            const { commentDiff } = await import('${{ github.workspace }}/.github/workflows/scripts/comments.mjs')
 
             // PR information
-            const number = '${{ github.event.pull_request.number }}'
+            const issueNumber = '${{ github.event.pull_request.number }}'
```

It was me that used strings (sorry) but looks like we've now switched to type number?

```suggestion
            const issueNumber = ${{ github.event.pull_request.number }}
```